src/lib/mongoBackend/dbConstants.h

```diff
@@ -110,6 +110,7 @@
 #define CSUB_HEADERS                 "headers"
 #define CSUB_QS                      "qs"
 #define CSUB_PAYLOAD                 "payload"
+#define CSUB_INCLUDEPAYLOAD          "includePayload"
```

Comment:
New field should be added to database model documentation. In particular to this section:

https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/database_model.md#csubs-collection

Reply:
Added